src/jsi/jsi_class.hpp

```diff
@@ -490,7 +490,9 @@ class ObjectWrap {
 
     static Internal* get_internal(JsiEnv env, const JsiObj& object)
     {
-        auto internal = object->getProperty(env, g_internal_field);
+        static const auto js_internal_field = fbjsi::String::createFromAscii(env, g_internal_field);
```

Comment:
This will break on reload since it will continue to use the string from the old env.

Reply:
Ah right yeah, good point. I guess there's no way round that? It's not a huge gain anyway so I'm happy to close this if that's best

Reply:
Make it a class-static like s_ctor with the same logic to set it on module load and reset on em teardown. This isnt a perfect solution because it assumes at most one env at a time, but it will at least work as well as the rest of the codebase.

Reply:
Updated in https://github.com/realm/realm-js/pull/4863/commits/e32fc5b82711ace47849baefcb0c836b48404653